src/components/KeypressListener/tests/KeypressListener.test.tsx

```diff
@@ -46,6 +52,37 @@ describe('<KeypressListener />', () => {
     listenerMap.keyup({keyCode: Key.Escape});
     expect(spy).not.toHaveBeenCalled();
   });
+
+  it('allows registering an event in the capture phase using useCapture', () => {
+    const spy = jest.fn();
+
+    mountWithApp(
+      <KeypressListener handler={spy} keyCode={Key.Escape} useCapture />,
+    );
+
+    expect(listenerOptionsMap.keyup).toBe(true);
+  });
+
+  it('passes additional options to the event', () => {
+    const spy = jest.fn();
+
+    const eventOptions = {
+      capture: true,
+      once: true,
+      passive: true,
+      signal: new AbortController().signal,
```

Comment:
I think this is totally unnecessary for this test and also for the `keyup|keydown` events, but I thought it would be interesting to show the full API here just in case. We can definitely simplify this to `{capture: true}` to make it simpler to read!